webserver/views/user.py

```diff
@@ -29,18 +29,20 @@ def lastfmscraper(user_id):
 
 @user_bp.route("/<user_id>")
 def profile(user_id):
-    cassandra = webserver.create_cassandra()
+    # Which database to use to showing user listens.
+    #db_conn = webserver.create_cassandra()
+    db_conn = webserver.create_postgres()
 
     # Getting data for current page
     max_ts = request.args.get("max_ts")
     if max_ts is not None:
         try:
-            max_ts = int(max_ts)
+            max_ts = int(float(max_ts))
```

Comment:
yeah. that is a good bug.
Python does not provide direct `string` to `int` conversion. It was failing when opening https://listenbrainz.org/user/armalcolite?max_ts=1459858239.0 i.e. when the timestamps are ending with decimals. (It throws on existing LB website as well)

On upgrading to postgres, the timestamp conversion ends with a decimal precision, and thus it was a problem. We could either reduce the precision in postgres, or do this to encorporate this change.